src/background/stores/global/user-configuration-store.ts

```diff
@@ -69,21 +69,21 @@ export class UserConfigurationStore extends BaseStoreImpl<UserConfigurationStore
     }
 
     @autobind
-    private onSetBugService(payload: SetBugServicePayload): void {
-        this.state.bugService = payload.bugServiceName;
+    private onSetIssueService(payload: SetIssueFilingServicePayload): void {
```

Comment:
Rename to `onSetIssueFilingService`? (There are about 60 instances of the string `IssueService` remaining in the code that may want to be checked.)

Reply:
When renaming things from IssueService to IssueFilingService I'm seeing weird behavior in jest tests where jest is not picking the new changes in tests. I and Wahaj are investigating this issue, since they are not really critical, can we move on with this PR?

Reply:
never mind. the phantom is gone.